modules/library/main/src/main/java/org/geotools/styling/SLDParser.java

```diff
@@ -1554,11 +1554,11 @@ protected ShadedRelief parseShadedRelief(Node root) {
                 childName = child.getNodeName();
             }
             if ("BrightnessOnly".equalsIgnoreCase(childName)) {
-                symbol.setBrightnessOnly(Boolean.getBoolean(getFirstChildValue(child)));
+                symbol.setBrightnessOnly(Boolean.parseBoolean(getFirstChildValue(child)));
```

Comment:
This part of the PR (on SLD parsing) looks unrelated. Is it meant to be included?

Reply:
`BrightnessOnly` is part of the shadedRelief specs.
The `getBoolean()` call is clearly a typo, since it retrieves the value of a system property, while the `parseBoolean()` will consider the text itself as a string to be parsed to a boolean value.